main.css

```diff
@@ -156,82 +89,51 @@
     color: #9EFFBC;
     text-shadow: 0px 0px 1px #BDBDBD;
 }
-
 .about > .nav-text {
     color: #ECD476;
     font-size: 22px;
 }
-
 .services > .nav-text {
     color: #8FF1FA;
     font-size: 22px;
 }
-
 .blog.diamond > .nav-text {
     color: #01CCFF;
     font-size: 22px;
     margin-top: 3px;
 }
-
 .contact > .nav-text {
     color: #BAD843;
     font-size: 22px;
     text-shadow: 1px 2px 10px #232323;
 }
-
-.image-height.text-body::-webkit-scrollbar-thumb {
-    background-color: #9A9A9A !important; 
-}
-
-.text-body.content-height::-webkit-scrollbar-track, 
-.with-scroll::-webkit-scrollbar-track {
-    background-color: transparent !important;
-}
-
-.text-body.content-height::-webkit-scrollbar, 
-.image-height.text-body::-webkit-scrollbar,
-.with-scroll::-webkit-scrollbar  {
+.image-height.text-body::-webkit-scrollbar-thumb { background-color: #9A9A9A !important; }
+.text-body.content-height::-webkit-scrollbar-track, .with-scroll::-webkit-scrollbar-track { background-color: transparent !important; }
+.text-body.content-height::-webkit-scrollbar, .image-height.text-body::-webkit-scrollbar, .with-scroll::-webkit-scrollbar  {
     width: 7px !important;
     background-color: transparent !important;
 }
-
 .text-body.content-height::-webkit-scrollbar-thumb,
 .with-scroll::-webkit-scrollbar-thumb {
     background-color: #D2D2D2 !important;
 }
-
 .with-scroll {
     overflow: auto;
     position: relative;
 }
-
-.no-padding {
-    padding: 0px !important;
-}
-
-.no-margin {
-    margin: 0px !important;
-}
-
+.no-padding { padding: 0px !important; }
+.no-margin { margin: 0px !important; }
 .first-image {
-    background-image: url('https://cdn.rawgit.com/dynobjx/dynobjx.github.io/master/01_Initial%20Image.png');
+    background-image: url('rsz_01_initial%20image.png');
```

Comment:
Change the directory of this image using the commit hash and not the master branch. Also use the rawgit cdn to optimize this image.